python/tvm/runtime/container.py

```diff
@@ -137,3 +137,24 @@ def __from_tvm_object__(cls, obj):
         val = str.__new__(cls, content)
         val.__tvm_object__ = obj
         return val
+
+
+@tvm._ffi.register_object("runtime.ShapeTuple")
+class ShapeTuple(Object):
```

Comment:
perhaps add __iter__?

Reply:
the iteration has been supported by `__getitem__`, see the test case